packages/ruby_rubocop.rb

```diff
@@ -6,43 +6,38 @@
   homepage 'https://rubocop.org'
-  version '1.36'
+  version '1.41.1-ruby-3.2'
   compatibility 'all'
   source_url 'SKIP'
-  # source_url 'https://github.com/rubocop/rubocop.git'
-  # git_hashtag "v#{version}"
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_armv7l/ruby_rubocop-1.26.1-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_armv7l/ruby_rubocop-1.26.1-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_i686/ruby_rubocop-1.26.1-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.26.1_x86_64/ruby_rubocop-1.26.1-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.41.1-ruby-3.2_armv7l/ruby_rubocop-1.41.1-ruby-3.2-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.41.1-ruby-3.2_armv7l/ruby_rubocop-1.41.1-ruby-3.2-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.41.1-ruby-3.2_i686/ruby_rubocop-1.41.1-ruby-3.2-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/ruby_rubocop/1.41.1-ruby-3.2_x86_64/ruby_rubocop-1.41.1-ruby-3.2-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '751d173e446bf5e478b5cf75d0d2bcf3ca67e8a0fbb602667c4e3d224e938c5c',
-     armv7l: '751d173e446bf5e478b5cf75d0d2bcf3ca67e8a0fbb602667c4e3d224e938c5c',
-       i686: '3ba763b890c2bb0a39cd07567bc248b7dc4b12242537e817ef1650b4fb2ce6bc',
-     x86_64: '59942e42753e15dafe07c9e522fa0e01483e5e2ca66b7c54a347bef8e58d4b8c'
+    aarch64: '5091d0c031c4e99a96c6276771f0f004a9b662b29d5a61c70a079203cf473bed',
+     armv7l: '5091d0c031c4e99a96c6276771f0f004a9b662b29d5a61c70a079203cf473bed',
+       i686: '08c568915137cbad80800158e5470721b55123260bde2fc886d358adaddb9cb2',
+     x86_64: '21b15de2bb144c053695fd00878d89e1fb30b5cfca8b51717f0a8615e519eb7d'
   })
 
-  no_fhs
-
   depends_on 'libyaml'
   depends_on 'ruby'
   depends_on 'xdg_base'
 
+  no_fhs
+
   @xdg_config_home = ENV.fetch('XDG_CONFIG_HOME', nil)
   @xdg_config_home = "#{CREW_PREFIX}/.config" if @xdg_config_home.to_s.empty?
 
-  def self.build
-    # system 'bundle install'
-    # system 'rake build'
-  end
+  def self.build; end
 
   def self.install
-    # system "gem install --build=#{CREW_DEST_DIR} pkg/rubocop-#{version}.gem"
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/.config/rubocop"
-    downloader 'https://github.com/chromebrew/chromebrew/raw/master/.rubocop.yml', 'SKIP', "#{CREW_DEST_PREFIX}/.config/rubocop/config.yml"
+    downloader 'https://github.com/chromebrew/chromebrew/raw/master/.rubocop.yml', 'SKIP',
+               "#{CREW_DEST_PREFIX}/.config/rubocop/config.yml"
```

Comment:
Move to `source_url` and `git_hashtag` instead?  Also, why isn't `.rubocop.yml` versioned?

Reply:
We would want a new repo just for this file then, otherwise it would require a git pull of the entire chromebrew repo just to get this one tiny file... that seems... wasteful.

Making that happen could be a separate issue which could be handled in its own time?

Reply:
Don't clone the repo, just download the file and sha256sum check it.

Reply:
hash added